src/__experimental__/components/simple-color-picker/simple-color-picker.d.ts

```diff
@@ -5,7 +5,7 @@ import { SimpleColorProps } from "./simple-color/simple-color";
 
 export interface SimpleColorPickerProps extends ValidationPropTypes, MarginProps {
   /** The SimpleColor components to be rendered in the group */
-  children: React.ReactElement<SimpleColorProps> | Array<React.ReactElement<SimpleColorProps>>;
+  children?: React.ReactElement<SimpleColorProps> | Array<React.ReactElement<SimpleColorProps>>;
```

Comment:
this makes children optional, but still disallows null, false and undefined as value in case of conditional rendering.

You need to define a type:
```
type SimpleColorPickerChildren = React.ReactElement<SimpleColorProps> | boolean | null | undefined;
```
and then use that type in children:
```
  children: SimpleColorPickerChildren | SimpleColorPickerChildren[];
```